### ISSUE TITLE
Unable to connect to trackers with successful VPN connection

### ISSUE THREAD

**Issue by ChangedNameTo** [haugene/docker-transmission-openvpn#1000]
I've been trying to debug this for around 2 days now before uploading this issue. I'm successfully connecting to NORDVPN, but have so far been unable to translate that into successful tracker responses. It was working for a bit then stopped working, and I haven't been able to diagnose why. I've disabled IPv6, run as root and as `pi`, no luck with anything yet. I'm able to ping 8.8.8.8 from root when I run commands, which makes this especially puzzling. Any advice would be great

**Docker Run**
```
docker run --cap-add=NET_ADMIN -d \
              -v /sharedfolders/downloads/:/data \
              -v /etc/localtime:/etc/localtime:ro \
              -v /home/pi/appdata/config:/config\
              -e CREATE_TUN_DEVICE=true \
              -e OPENVPN_PROVIDER=NORDVPN \
              -e OPENVPN_USERNAME=blah \
              -e OPENVPN_PASSWORD=blah \
              -e WEBPROXY_ENABLED=false \
              -e LOCAL_NETWORK=192.168.0.0/16 \
              -e OPENVPN_OPTS="--inactive 3600 --ping 10 --ping-exit 60" \
              --log-driver json-file \
              --log-opt max-size=10m \
              --name transmission \
              -p 9091:9091 \
              haugene/transmission-openvpn:latest-armhf
```

**Logs**
[transmission.log](https://github.com/haugene/docker-transmission-openvpn/files/4037023/transmission.log)

**Host system:**
Raspberry Pi

**Comment by ChangedNameTo** [comment]
I've realized that a lot of the log was bloated with files that were completed yet still in resume/torrents, clearing that out has filtered out a lot of the noise but still hasn't resolved the issue at hand.

**Comment by eximo84** [comment]
My docker installation has also started this using Mullvad VPN.  I have PIA currently also that i am having similar issues with that too.

**Comment by haugene** [comment]
Can you try to ping a DNS address instead of 8.8.8.8? Like google.com
Hoping this is just a dns issue and that you need to add --dns options to your run command.

**Comment by eximo84** [comment]
Update from me, my issues related to Mullvad provider not providing a port.  I had to create one in my Mullvad account then update the port within Transmission.  I was also using a random OVPN profile for France UDP which seems to not work so ive reset this back to default.ovpn and things are now working for me as they should.

**Comment by ChangedNameTo** [comment]
I resolved this late last night. The issue was from transmission's config directory being pointed at my NAS, and it didn't like that very much. To resolve, I removed the container, cleared out all of the files in /resume and /torrents, rebooted the docker pointing at a local config directory.

For reference this issue will allow you to **still** ping google even if the torrents are not resolving.

Sorry to bother! Hope this helps someone in the future.
